set.mm

```diff
@@ -407922,11 +408076,156 @@ sequence p(0) p(1) ... p(n) of the vertices in a walk p(0) e(f(1)) p(1)
+      eqtr3 com12 imp simpr 1e2m1 a1i oveq2d 2cnd 1cnd addsubassd eqtr4d eqeq2d
+      nn0cn mpbird oveq2 eqeq12d syl biimpd com13 com23 impcom 3ad2ant2 3adant3
+      ex imp31 c0 wne simpl anim12i cc0 clt wbr nn0re 2re nn0ge0 2pos addgegt0d
+      breq2 hashgt0n0 sylan2 exp32 jca32 hashneq0 biimprd pfxsuff1eqwrdeq ancom
+      cr syl3anc anbi2i 3anass bitr4i syl6bb mpbir3and exp31 syl2anb ralrimivva
+      sylbid dff13 sylanbrc ) IUCUDZCDGUEUAUFZGUGZUBUFZGUGZQZYQYSQZRZUBCUHUACUH
+      CDGULABCDEFGHIJKLMNOPUIYPUUCUAUBCCYPYQCUDZYSCUDZSZSUUAYQUJUGZYSUJUGZQZUUB
+      UUFUUAUUIUMYPUUDUUEYRUUGYTUUHBYQBUFZUJUGZUUGCGUUJYQUJUKPYQUJUNUOBYSUUKUUH
+      CGUUJYSUJUKPYSUJUNUOUPUQUUFYPUUIUUBRZUUDYQJURZUDZYQUSUGZIUTVAVBZQZYQIVCVA
+      VBZVFVBZKQZKUJUGZUUGVDZFUDZVGZSZYSUUMUDZYSUSUGZUUPQZYSUURVFVBZKQZUVAUUHVD
+      ZFUDZVGZSZYPUULRUUEAUFZUSUGUUPQZUVOUURVFVBZKQZUVAUVOUJUGZVDZFUDZVGZUVDAYQ
+      UUMCUVOYQQZUVPUUQUVRUUTUWAUVCUVOYQUUPUSVEUWCUVQUUSKUVOYQUURVFVHVIUWCUVTUV
+      BFUWCUVSUUGUVAUVOYQUJUKVJVKVLNVMUWBUVMAYSUUMCUVOYSQZUVPUVHUVRUVJUWAUVLUVO
+      YSUUPUSVEUWDUVQUVIKUVOYSUURVFVHVIUWDUVTUVKFUWDUVSUUHUVAUVOYSUJUKVJVKVLNVM
+      UVEUVNSZYPUUIUUBUWEYPSZUUISZUUBUUOUVGQZUUIYQUUOVCVNVBZVFVBZYSUWIVFVBZQZUW
+      FUWHUUIUWEUWHYPUVEUVNUWHUVDUVNUWHRZUUNUUQUUTUWMUVCUVNUUQUWHUVMUUQUWHRZUVF
+      UVHUVJUWNUVLUUQUVHUWHUUOUVGUUPVQVOVPUQVRVPUQVSTTUWFUUIVTUWFUWLUUIUVEUVNYP
+      UWLUVDUVNYPUWLRZRZUUNUUQUUTUWPUVCUVNUUQUUTSZUWOUVMUWQUWORZUVFUVJUVHUWRUVL
+      UWQUVJUWOUUTUUQUVJUWORUUTUVJUUQUWOUUTUVJUUQUWORZUUTUVJSUUSUVIQZUWSUUSUVIK
+      VQYPUUQUWTUWLYPUUQUWTUWLRYPUUQSZUWTUWLUXAUURUWIQZUWTUWLUMUXAUXBUURUUPVCVN
+      VBZQZYPUXDUUQYPUURIUTVCVNVBZVAVBUXCYPVCUXEIVAVCUXEQYPWAWBWCYPIUTVCIWIYPWD
+      YPWEWFWGTUUQUXBUXDUMYPUUQUWIUXCUURUUOUUPVCVNVHWHUQWJUXBUUSUWJUVIUWKUURUWI
+      YQVFWKUURUWIYSVFWKWLWMWNWTWOWMWTWPWQVRWRUQVRWSUQXATUWGUUNUVFSZYQXBXCZYSXB
+      XCZSZSZUUBUWHUUIUWLVGZUMUWFUXJUUIUWFUXFUXGUXHUWEUXFYPUVEUUNUVNUVFUUNUVDXD
+      UVFUVMXDXETUWEYPUXGUVEYPUXGRZUVNUVDUUNUXLUUQUUTUUNUXLRUVCUUNUUQUXLUUNUUQY
+      PUXGUUQYPSZUUNXFUUOXGXHZUXGUXMUXNXFUUPXGXHZYPUXOUUQYPIUTIXIUTYCUDYPXJWBIX
+      KXFUTXGXHYPXLWBXMZUQUUQUXNUXOUMYPUUOUUPXFXGXNTWJYQUUMXOXPXQVRVPWQTVSUWEYP
+      UXHUVNYPUXHRZUVEUVMUVFUXQUVHUVJUVFUXQRUVLUVFUVHUXQUVFUVHYPUXHUVHYPSZUVFXF
+      UVGXGXHZUXHUXRUXSUXOYPUXOUVHUXPUQUVHUXSUXOUMYPUVGUUPXFXGXNTWJYSUUMXOXPXQV
+      RVPWQUQVSXRTUXJUUBUWHUWLUUISZSZUXKUXJUUNUVFUXNUUBUYAUMUXFUUNUXIUUNUVFXDTU
+      XFUVFUXIUUNUVFVTTUXIUXFUXNUXGUXFUXNRUXHUXFUXGUXNUUNUXGUXNRUVFUUNUXNUXGYQU
+      UMXSXTTVRTWQYSJYQYAYDUYAUWHUUIUWLSZSUXKUXTUYBUWHUWLUUIYBYEUWHUUIUWLYFYGYH
+      WMYIYJYKWQYMYLUAUBCDGYNYO $.
+
+    $d D d r $.  $d E d $.  $d F r $.  $d G d r $.  $d N d n r t w $.
+    $d R d r $.  $d V d $.  $d W d i r $.
+    $( Lemma for ~ wwlksnextbij .  (Contributed by Alexander van der Vekens,
+       7-Aug-2018.)  (Revised by AV, 18-Apr-2021.)  (Revised by AV,
+       27-Oct-2022.) $)
+    wwlksnextsur $p |- ( W e. ( N WWalksN G ) -> F : D -onto-> R ) $=
```

Comment:
I would label it "wwlksnextsurj"

Reply:
OK (so that it is not confused with surreal numbers): Done